.github/workflows/build.yaml

```diff
@@ -375,3 +375,38 @@ jobs:
 
       - name: Run codespell
         run: codespell -q 7 -S ".git,build*,./src/third_party/*" -I misc/codespell-allowlist.txt
+
+  deploy:
+    name: Deploy
+    runs-on: windows-2019
+    strategy:
+        fail-fast: false
+    steps:
+      - name: Get source
+        uses: actions/checkout@v2
+
+      - name: Prepare Windows environment (Visual Studio)
+        if: runner.os == 'Windows'
```

Comment:
The check is isn't needed, as it's already specified to run on windows-2019.

Reply:
Removed superfluous code